src/main/java/io/jenkins/plugins/designlibrary/Links.java

```diff
@@ -28,7 +28,8 @@ public ContextMenu doContextMenu(StaplerRequest request, StaplerResponse respons
             // otherwise you can also programatically create them.
             // see the javadoc for various convenience methods to add items
             return new ContextMenu()
-                    .add(new MenuItem().withContextRelativeUrl("/").withIconClass("icon-up").withDisplayName("Back to Dashboard"))
+                    .add(new MenuItem().withContextRelativeUrl("/").withIconClass("symbol-arrow-up-outline plugin-ionicons-api")
+                            .withDisplayName("Back to Dashboard"))
```

Comment:
Works now thanks, I think it would be better to remove this though as it's quite a weird one, you could always stick icons on the other ones instead.

Jenkins icon + a docs sort of one

Reply:
Thought the same, as linking "up" or to "dashboard" in sidepanel is also not recommended.